Ignore Horizon routes

adampatterson opened the pull request:
I noticed that the routes for Telescope were ignored and thought that it would also be good to exclude Horizons routes.

zepfietje commented:
Thanks, @adampatterson, I agree it would be good to also ignore Horizon routes by default.
However, as the list of routes to be ignored gets longer, maybe we should think of a configurable solution? Any ideas?

adampatterson commented:
@zepfietje   I like that, you may not want to track dashboard or admin views.

I'll update my PR over the next few days to pull from an array in the pirsch config file.

zepfietje commented:
Not tracking dashboard or admin views is quite easy already by just removing (or not adding) the middleware from those routes. So this feature would be most useful for third-party package routes. But yes, might as well use it to simplify ignoring a group of routes.

Let me know when you're ready by marking this PR as ready for review. 😊

Autive commented on src/Http/Middleware/TrackPageview.php:
```diff
@@ -21,8 +21,10 @@ public function handle(Request $request, Closure $next): mixed
             return $response;
         }
 
-        if (str_starts_with($request->route()->uri, 'telescope/')) {
```
Wouldn't this be a bit cleaner?

```php
		if ( in_array( $request->route()->uri, config( 'pirsch.excluded_routes' ) ) ) {
			return $response;
		}
```

zepfietje commented:
Is this ready for review, @adampatterson? 🙂

adampatterson commented:
@zepfietje Iet me double check this weekend.

I had Pirsch running on a server that Oracle decided to delete on me 🙃

zepfietje commented:
No rush, @adampatterson! Just wondered what the status of this PR was :)

zepfietje commented:
Replaced by https://github.com/pirsch-analytics/laravel-pirsch/pull/9.

adampatterson commented:
> Replaced by #9.

Sorry, I no longer had a way of testing with Pirsh.

zepfietje commented:
No worries, @adampatterson.

adampatterson commented:
@zepfietje what's the best method for testing Pircsh since the trial is limited to 30 days?

zepfietje commented:
I'm sure @Kugelschieber can help you test it for a little longer if needed. 🙂

Kugelschieber commented:
@adampatterson I can extend your trial. Can you send an email to support@pirsch.io with your account email address?

adampatterson commented:
@zepfietje I went ahead and updated my branch with the latest changes and tested in Pirsh and everything seems to be working as expected but I did notice something odd with the unit tests.

I have never used PEST before so duplicated a few previous tests and set the configs as if I were going to exclude a custom route or header.

https://github.com/adampatterson/laravel-pirsch/blob/middleware/tests/TrackPageviewTest.php#L50

And they passed, so naturally I tried to break the test and it kept passing.

I then went back to a previous test and modified the `excluded_routes` values and the test still passed.

I then modified the test it's self and it kept passing.

```php
return [
    'excluded_routes' => [
        //
    ],
,
```

```php
it('skips Telescope', function () {
    Pirsch::spy();

    Route::middleware(TrackPageview::class)
        ->get('thisshouldcall/test', fn() => 'Hello World');

    $this->get('/thisshouldcall/test');

    Pirsch::shouldNotHaveBeenCalled();
});
```

I would expect this to fail since the route is not excluded.
